packages/composite/src/__package__.ts

```diff
@@ -0,0 +1,2 @@
+export const PKG_NAME = "@hpcc-js/composite";
+export const PKG_VERSION = "2.0.21";
```

Comment:
Can't we just parse this info from each package's `package.json` file at build time and store it globally? Or within `common_Widget`?

Reply:
But you might have a newer version of "map" using an older version of "common" and it would screw things up (unlikely I know).
But I do think storing the loader version here will be a good idea (the loader version implicitly defines all the modules versions as well)